src/runtime/tests/reference-test.ts

```diff
@@ -54,7 +54,7 @@ describe('references', () => {
 
   it('exposes a dereference API to particles', async () => {
     const loader = new StubLoader({
-      manifest: `
+      'manifest': `
```

Comment:
there is another unquoted manifest in a StubLoader constructor in arc-test.ts and other unquoted strings in manifest-test.ts - why does it break here?

Reply:
Oh, sorry, this was just an unrelated cleanup.

Reply:
The root cause of the breakage was that stubloader async caused something in Arc to be unresolved.  Adding the arc.idle call fixed it.